backend/src/main/resources/graphql/main.graphqls

```diff
@@ -253,6 +254,11 @@ type TestResult {
   patientLink: PatientLink
 }
 
+type MultiplexResult {
```

Comment:
I think this can probably be removed - I'm guessing it's the result of an auto-merge, since you replaced this with DiseaseResult

Reply:
I actually didn't change the name of this type (I only changed the name of the helper class in the backend in the previous PR, this is a new GraphQL type) - I think I need to rename this type to DiseaseResult, as it's used on line 248. Good catch!

Reply:
Ohhh nvm, I already have an input called DiseaseResult on line 303, but I need this type separately for the view query, any thoughts on naming?

Reply:
Ahh, I see. Discussed offline - keeping it as `Multiplex Result` works for me!